app/views/DR/QRview/index.js

```diff
@@ -0,0 +1,62 @@
+import { Button, Text } from 'native-base';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Platform, View } from 'react-native';
+import QRCode from 'react-native-qrcode-svg';
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen';
+import Share from 'react-native-share';
+
+import NavigationBarWrapper from '../../../components/NavigationBarWrapper';
+import buttonStyle from '../../../constants/DR/buttonStyles';
+
+export default function QRview({ navigation }) {
+  const { t } = useTranslation();
+  const shopUrl =
+    Platform.OS === 'ios'
+      ? 'https://apps.apple.com/do/app/covid-rd/id1506071684?l=en'
+      : 'https://play.google.com/store/apps/details?id=com.optic.covdr';
+  const backToMain = () => {
+    navigation.goBack();
+  };
+
+  const openShareScreen = () => {
+    const shareOptions = {
+      type: 'text',
+      title: '',
+      url: shopUrl,
+      failOnCancel: false,
+    };
+
+    Share.open(shareOptions)
+      .then(res => console.log(res))
+      .catch(err => console.error(err));
```

Comment:
Async Await function maybe? For better consistence in the project. A personal opinion, Isn't necessary.